server/src/completionUtil.ts

```diff
@@ -63,119 +67,219 @@ export const blockAttributes: CompletionItem[] = [
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@map(_ name: String)',
+        '```',
+        '___',
+        'Maps a field name from the Prisma schema to a different column name.',
+        '',
+        '_@param_ `name` The name of the target database column',
+      ].join('\n'),
+    },
   },
   {
     label: '@default()',
     kind: CompletionItemKind.Property,
-    detail: '@default(_ expr: Expr)',
     insertTextFormat: 2,
     insertText: '@default(0)',
-    documentation: 'Specifies a default value if null is provided.',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@default(_ expression: Expression)',
+        '```',
+        '___',
+        'Defines a default value for this field. `@default` takes an expression as an argument.',
+        '',
+        '_@param_ `expression` An expression (e.g. `5`, `true`, `now()`)',
+      ].join('\n'),
+    },
   },
   {
     label: '@relation()',
     kind: CompletionItemKind.Property,
     insertTextFormat: 2,
     insertText: '@relation($0)',
-    detail:
-      '@relation(_ name?: String, references?: Identifier[], onDelete?: CascadeEnum)\nArguments:\n•name: (optional, except when required for disambiguation) defines the name of the relationship. The name of the relation needs to be explicitly given to resolve amibiguities when the model contains two or more fields that refer to the same model (another model or itself).\n•references: (optional) list of field names to reference',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: [
+        '```prisma',
+        '@relation(_ name: String?, fields: FieldReference[]?, references: FieldReference[]?)',
+        '```',
+        '___',
+        'Defines meta information about the relation. [Learn more](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/relations#the-relation-attribute).',
+        '',
+        '_@param_ `name` A name of field references',
+        '',
+        '_@param_ `fields` A list of field references',
+        '',
+        '_@param_ `references` A list of field references',
+      ].join('\n'),
+    },
+  },
+  {
+    label: '@updatedAt',
+    kind: CompletionItemKind.Property,
     documentation:
-      'Specifies and disambiguates relationships when needed. Where possible on relational databases, the @relation annotation will translate to a foreign key constraint, but not an index.',
+      'Automatically stores the time when a record was last updated.',
   },
 ]
 
 export const supportedDataSourceFields: CompletionItem[] = [
   {
     label: 'provider',
     kind: CompletionItemKind.Field,
-    documentation:
-      'Can be one of the following built in datasource providers:\n•`postgresql`\n•`mysql`\n•`sqlite`',
+    documentation: {
+      kind: MarkupKind.Markdown,
+      value: 'Describes which data source connector to use.',
```

Comment:
Is that wording we use anywhere else?

Wouldn't having the valid value here be nice?

Reply:
This is the wording used in Prisma Docs. https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-schema/data-sources/#fields

Reply:
But I agree, the valid value would be nice here. I'll add that.